contracts/ROSCA.sol

```diff
@@ -6,19 +6,43 @@ pragma solidity ^0.4.4;
  * The winner of the pot is decided by reverse auction (lowest Bid wins).
  *
  * Things still missing/needs fixing:
- * in withdraw() , add fee related logic
- * checkingContribution, check whether or not a member as contributed as least contributionSize at the end of Round
+ * in withdraw() , add fee related logic (done, Sol should still be discussed)
+ *   Sol: will keep deduct a fee, and keep it in contract untill the end?
+ * if there are contribution than what is required, put them in pendingWithdrawal (done)
+ *   Sol: in contribute , check if contributed is enough to last the whole ROSCA, if it is put the excess in pendingWithdrawal
+ * when the rosca ends all the money should go out to the account user (done, still needs to work out sending fees to wetrust address)
+ *   Sol: added endOfROSCA, cleanUp and sendFUND function
+ * checkingContribution, check whether or not a member as contributed as least contributionSize at the end of Round (goodStanding var takes care of this i think)
+ *  -startRound should check?
+ * if user doesnt contribute , dont allow bidding EVER! have to be in good standing to bid/win the Pot (done)
+ *   Sol: used goodStanding variable to keep track of how many round a person can bet
+ * contribute and withdraw doesnt check if round has started (done)
+ *
+ * Product Decision to think about:
+ *   - biding when not everyone had contributed, total contributed is less than the bid amount
+ *     Possible Solution:
+ *       - owed variable? before paying the next round winner, owed is paid first?
+ *       - startRound Check ? if balance is less than lowest Bid ? if it is, what to do?
+ *       - in withdraw, check balance of contract first and if its less than pending withdrawal, subtract that amount only (I think this is the simplest option) (implement for now)
+ *
+ * Things brought up that needs considering:
+ *   - allowing constructor to have initial list of members specified by foreman on ROSCA deployment
+ *   - using a better unit than wei to represent ether i.e finney (we shouldn't be dealing with any value less than finney anyways)
+ *   - getters for necessary variables (making variables public would work)
```

Comment:
havent given much thought, on the top of my head, currentRound, lowestBid, members